### PR TITLE
Minor updates to reflect latest changes in storm and some additional convenience functions

### PR DESCRIPTION
**Pull request by sjunges**

### CODE REVIEW

File: lib/stormpy/__init__.py [hunk 1]
```diff
@@ -481,12 +493,36 @@ def topological_sort(model, forward=True, initial=[]):
...
+    Get the states that are reachable in a sparse model
+
+    :param model: A model
+    :param initial_states:
+    :param constraint_states:
+    :param target_states:
+    :param maximal_steps:
+    :param choice_filter:
+    :return:
+    """
```
Comment (volkm) [inline lib/stormpy/__init__.py:507]: Add explanations for arguments?

File: src/core/modelchecking.cpp [hunk 2]
```diff
@@ -88,10 +95,22 @@ void define_modelchecking(py::module& m) {
...
+
     m.def("_get_reachable_states_double", &getReachableStates<double>, py::arg("model"), py::arg("initial_states"), py::arg("constraint_states"), py::arg("target_states"), py::arg("maximal_steps") = boost::none, py::arg("choice_filter") = boost::none);
     m.def("_get_reachable_states_exact", &getReachableStates<storm::RationalNumber>, py::arg("model"), py::arg("initial_states"), py::arg("constraint_states"), py::arg("target_states"), py::arg("maximal_steps") = boost::none, py::arg("choice_filter") = boost::none);
     m.def("_get_reachable_states_rf", &getReachableStates<storm::RationalFunction>, py::arg("model"), py::arg("initial_states"), py::arg("constraint_states"), py::arg("target_states"), py::arg("maximal_steps") = boost::none, py::arg("choice_filter") = boost::none);
 
+    m.def("_compute_expected_number_of_visits_double", &getExpectedNumberOfVisits<double>, py::arg("env"), py::arg("model"));
+    m.def("_compute_expected_number_of_visits_double", &getExpectedNumberOfVisits<storm::RationalNumber>,  py::arg("env"), py::arg("model"));
```
Comment (volkm) [inline src/core/modelchecking.cpp]: _compute_expected_number_of_visits_exact

**Review by volkm — APPROVED** [review]
LGTM. Just minor comments.

**Comment by volkm** [comment]
I am a bit confused why commits from the main repo are also occurring here. Maybe something is not entirely perfect with the history?

File: src/core/modelchecking.cpp (hunk 2 above)
Comment (sjunges) [reply src/core/modelchecking.cpp]: oops, thanks

**Comment by sjunges** [comment]
Yes; I think this is due to a rebase

**Pull request merged**